src/webring.rs

```diff
@@ -59,12 +64,12 @@ impl Member {
 
 #[derive(Clone, Debug, Default)]
 struct WebringData {
-    members_table: HashMap<Authority, usize>,
+    members_table: HashMap<Intern<str>, usize>,
```

Comment:
Why `Intern<str>` here when everything else uses `Intern<Uri>`? If it's because it's storing just the authority and not the whole URI, we could still use `Intern<Authority>`.

Reply:
Good point; I use them to store authorities, but there was a reason that I couldn't use `Intern<Authority>`. I think it was that `Authority` doesn't implement `Serialize`, but I cut the functionality that needed that. I will try to refactor in the morning...

Reply:
NVM I did it now